st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -134,8 +134,8 @@
                 'secret': True
             },
             'private_key': {
-                'description': ('Private key material to log in. Note: This needs to be actual '
-                                'private key data and NOT path.'),
+                'description': ('Private key material or path to the private key file on disk '
```

Comment:
Added docs which clarify the requirements and potential security risks when using this feature - https://github.com/StackStorm/st2docs/pull/147.